api/index.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Route Newspack Campaigns API requests based on method.
+ *
+ * @package Newspack
+ */
+
+switch ( $_SERVER['REQUEST_METHOD'] ) { //phpcs:ignore
```

Comment:
This should be good enough for a 0.1 version, very simple and efficient. Bat later on, we could improve it with some light-weight and more functional routing.

Reply:
Yes, although this particular endpoint is extremely simple - there are only two variants and should never be more.